guides/channels/channels.md

```diff
@@ -1,4 +1,4 @@
-# Introduction
```

Comment:
Should I update the file path from `guides/channels/channels.md` to be something like `guides/real-time/channels.md`? Just to follow the pattern described in the sidebar.

Reply:
I would do so. As it's easier to find `presence.md` then.
But wait for some of the maintainers to approve this oppinion.